Configuration/ProcessModifiers/python/useMVASel_cff.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+
+# Modifier to enable the use of the MVA selection on 
+# tracks for the 4D vertex reco
+
+useMVASel =  cms.Modifier()
```

Comment:
I think we should use a less generic name for this process modifier. 
Something like `vertex4dTrackSelMVA` or something.